docs/en/compute/HBase.md

```diff
@@ -122,8 +122,17 @@ After this mapreduce job finishes, you can see a result like this:
 
 ### Alluxio in HA mode
 
-When Alluxio is running in fault tolerant mode, change the `hbase.rootdir` property in `conf/hbase-site.xml`
-to include Zookeeper information.
+When Alluxio is running in HA mode with Embedded Journal, change the `hbase.rootdir` property
```

Comment:
instead of "running **with** Embedded Journal" how do you feel about "configured to use the embedded journal"?

Reply:
Embedded journal will be the default journal type soon, `configured to use the embedded journal` sounds like we need to configured to enable Embedded journal feature, but I agree that running with Embedded Journal sounds a little strange since it's an internal feature.
What about `Running in embedded journal HA mode`?